packages/storybook/index.html

```diff
@@ -7,13 +7,29 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <title>Component Test Page</title>
   <script src="/src/index.ts" type="module"></script>
+  <!--
+    the following file is public/global.css, it will not have postcss processing
+    run on it and will be copied as is into the dist directory upon a build.
+  -->
   <link rel="stylesheet" href="/global.css" />
+  <!--
+    the following files have to be referenced individually for development but roll up into a single style.css
+    file in the built package, see comments in src/index.ts for more info and to control the files bundled.
+    These files are processed by postcss.
+
```

Comment:
It may not be true that these need to be referenced here as long as they are imported in `src/index.ts` based on my latest testing. I think it would be good to mention in this file that `style.css` does need to be included by normal consumers (assuming that is true and `dist/index.js` isn't going to also import style.css).